Fitness/Controllers/HomeViewController+Extensions.swift

```diff
@@ -0,0 +1,224 @@
+//
+//  HomeViewController+Extensions.swift
+//  Fitness
+//
+//  Created by Kevin Chan on 5/20/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+extension HomeViewController: UICollectionViewDataSource {
+
+    func numberOfSections(in collectionView: UICollectionView) -> Int {
+        return sections.count
+    }
+
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return 1
+    }
+
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let sectionType = sections[indexPath.section]
+        switch sectionType {
+        case .checkIns:
+            // swiftlint:disable:next force_cast
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: Constants.checkInsListCellIdentifier, for: indexPath) as! CheckInsListCell
+            cell.configure(for: habits)
+            return cell
+        case .allGyms:
+            // swiftlint:disable:next force_cast
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: Constants.gymsListCellIdentifier, for: indexPath) as! GymsListCell
+            cell.delegate = self
+            cell.configure(for: self.favoriteGyms)
+            return cell
+        case .todaysClasses:
+            // swiftlint:disable:next force_cast
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: Constants.todaysClassesListCellIdentifier, for: indexPath) as! TodaysClassesListCell
+            cell.delegate = self
+            cell.configure(for: gymClassInstances)
+            return cell
+        default:
+            // swiftlint:disable:next force_cast
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: Constants.prosListCellIdentifier, for: indexPath) as! ProsListCell
+            cell.delegate = self
+            cell.configure(for: pros)
+            return cell
+        }
+    }
+
+    func collectionView(_ collectionView: UICollectionView, viewForSupplementaryElementOfKind kind: String, at indexPath: IndexPath) -> UICollectionReusableView {
+        // swiftlint:disable:next force_cast
+        let headerView = collectionView.dequeueReusableSupplementaryView(ofKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: HomeSectionHeaderView.identifier, for: indexPath) as! HomeSectionHeaderView
```

Comment:
this line seems excessively long, does SwiftLint have limits on how long lines should be?